xio-core/src/main/java/com/xjeffrose/xio/http/DefaultFullRequest.java

```diff
@@ -71,9 +66,21 @@ public Builder host(String host) {
       headers().get().set(HttpHeaderNames.HOST, host);
       return this;
     }
+
+    abstract Optional<TraceInfo> httpTraceInfo();
+
+    abstract DefaultFullRequest autoBuild();
+
+    public DefaultFullRequest build() {
+      if (!httpTraceInfo().isPresent()) {
+        httpTraceInfo(new TraceInfo());
+      }
+      streamId(-1);
```

Comment:
Please put this back into `builder` as it is a default there and a constant here.

Reply:
done